app/test/unit/create-branch-test.ts

```diff
@@ -42,6 +44,8 @@ const someOtherBranch: Branch = {
   upstreamWithoutRemote: null,
   nameWithoutRemote: 'some-other-branch',
   isDesktopForkRemoteBranch: false,
+  ref: '',
```

Comment:
Same: if you agree with my suggestion above:
```suggestion
  ref: null,
```